apps/www/app/(main)/door/[code]/(private)/exchanges/inbox/@navbar/_client/FilterForm.tsx

```diff
@@ -67,15 +67,22 @@ const search_form_classes = tv({
   base: `grid grid-cols-2 gap-4`,
   slots: {
     button: ui_button({
-      className: "whitespace-nowrap",
-      intent: "whity",
+      className: "w-1/2 whitespace-nowrap",
```

Comment:
```suggestion
      className: "w-1/2 whitespace-nowrap",
```

Can you find another way ? The button is an inline-block right ?

Reply:
Flex-1 Done. Does it suits you ?
#FLEXFORLIFE